src/Polly.Core/Utils/CancellationTokenSourcePool.Pooled.cs

```diff
@@ -0,0 +1,42 @@
+namespace Polly.Utils;
+
+internal abstract partial class CancellationTokenSourcePool
+{
+#if NET6_0_OR_GREATER
+    private sealed class PooledCancellationTokenSourcePool : CancellationTokenSourcePool
+    {
+        public static readonly PooledCancellationTokenSourcePool SystemInstance = new(TimeProvider.System);
```

Comment:
Do we need the `System` prefix?

Reply:
Yeah, it collides with `Polly.Timeout` :/ It's super annoying.